Mathlib/FieldTheory/SeparableDegree.lean

```diff
@@ -18,78 +18,108 @@ This file contains basics about the separable degree of a field extension.
 
 ## Main definitions
 
-- `Field.Emb F E`: the type of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`.
+- `Field.Emb F E`: the type of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`
+  (mathematically, it should be the algebraic closure of `F`, but in order to make its cardinality
+  has the same type with `Module.rank F E`, we use the algebraic closure of `E`).
```

Comment:
maybe
```suggestion
- `Field.Emb F E`: the type of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`
  (the algebraic closure of `F` is usually used in the literature, but our definition has the advantage
  that `Field.Emb F E` lies in the same universe as `E` rather than the maximum over `F` and `E`).
```